### PR TITLE
JBIDE-27067 Add JSF 2.3 in new project wizard

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

**Comment by jkopriva** [comment]
Should be merged after: https://github.com/jbosstools/jbosstools-base/pull/657

File: jsf/plugins/org.jboss.tools.jsf.ui/src/org/jboss/tools/jsf/ui/operation/JSFProjectCreationOperation.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ protected void configFacets(IDataModel dataModel, String projectLocation) {
 					: template.indexOf("2.0") >= 0 ? "2.0"
 					: template.indexOf("2.1") >= 0 ? "2.1"
 					: template.indexOf("2.2") >= 0 ? "2.2"
+					: template.indexOf("2.2") >= 0 ? "2.3"
```
Comment (jeffmaury) [inline jsf/plugins/org.jboss.tools.jsf.ui/src/org/jboss/tools/jsf/ui/operation/JSFProjectCreationOperation.java]: Shouldn't it be template.indexOf("2.3") ?

Comment (jkopriva) [reply jsf/plugins/org.jboss.tools.jsf.ui/src/org/jboss/tools/jsf/ui/operation/JSFProjectCreationOperation.java]: Thanks!

Comment (jkopriva) [reply jsf/plugins/org.jboss.tools.jsf.ui/src/org/jboss/tools/jsf/ui/operation/JSFProjectCreationOperation.java]: Fixed

**Comment by jkopriva** [comment]
@jeffmaury can you please review it once again? Thank you!

**Pull request merged**